app/gui/src/controller/searcher/component.rs

```diff
@@ -54,6 +56,29 @@ pub enum Order {
 
 
 
+// ============
+// === Kind ===
+// ============
+
+/// The kind of a [`Component`].
```

Comment:
This is a type `Kind` in a module `component`, so this is obvious that this is "a kind of a component". This doc tells obvious thing, but does not explain at all what this thing really is.

I've got a very similar request to previou PR's - please go torugh ALL doc comments, and make them better. This is the only problem I see with this and many of your previous PRs. Ping me when it will be done, I'll check it then again (I'm stopping checking the code now, as there was already several places with non-clear docs up to this point).